sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -67,12 +74,12 @@ export interface EventHubSenderOptions {
  * Describes the EventHubSender that will send event data to EventHub.
  * @internal
  */
-export class EventHubSender extends LinkEntity {
+export class EventHubSender {
   /**
    * The unique lock name per connection that is used to acquire the
    * lock for establishing a sender link by an entity on that connection.
    */
-  readonly senderLock: string = `sender-${uuid()}`;
+  private readonly senderLock: string = `sender-${uuid()}`;
```

Comment:
seems like you could use your new `getRandomName` helper here?

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/25214/commits/9e1faf1551c63799b5ad35689ebb081699067fa1.